gradle.template.properties

```diff
@@ -8,3 +8,6 @@ org.gradle.daemon=false
 # e.g org.gradle.java.home=/Library/Java/JavaVirtualMachines/jdk1.8.0_144.jdk/Contents/Home
 # e.g org.gradle.java.home=C:/Program Files/Java/jdk1.8.0_144
 # org.gradle.java.home=
+
+# The test environment REQUIRES the following locale. Do not change!
+org.gradle.jvmargs=-Duser.language=en -Duser.country=US
```

Comment:
You should remove this?

Reply:
Yes. This workaround is no longer necessary :)